src/track/trackrecord.cpp

```diff
@@ -47,4 +49,88 @@ bool TrackRecord::updateGlobalKeyText(
     return false;
 }
 
+namespace {
+
+void mergeReplayGainMetadataProperty(
```

Comment:
I'm getting a warning now:
```
/home/jan/Projects/mixxx/src/track/trackrecord.cpp:54:6: Warnung: »void mixxx::{anonymous}::mergeReplayGainMetadataProperty(mixxx::ReplayGain&, const mixxx::ReplayGain&)« definiert, aber nicht verwendet [-Wunused-function]
   54 | void mergeReplayGainMetadataProperty(
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Oh, damn. An #ifdef is missing. Fix will follow.